tests/e2e/vLLM/test_vllm.py

```diff
@@ -146,3 +163,35 @@ def test_vllm(self):
     def tear_down(self):
         if self.save_dir is not None:
             shutil.rmtree(self.save_dir)
+
+    def _check_session_contains_recipe(self) -> None:
```

Comment:
move to utils

Reply:
YOu only this func here, its not shared func. No need to move it